src/ray-input.js

```diff
@@ -23,12 +23,13 @@ import InteractionModes from './ray-interaction-modes'
  * API wrapper for the input library.
  */
 export default class RayInput extends EventEmitter {
-  constructor(camera, opt_el) {
+  constructor(camera, opt_el, vrDisplay) {
```

Comment:
If we do need the extra param, could we make the opt_el parameter be the last one?